src/pages/Graph/SummaryPanel.tsx

```diff
@@ -16,11 +16,13 @@ type MainSummaryPanelPropType = SummaryPanelPropType & {
 };
 
 const expandedStyle = style({
+  fontSize: '74%', // TODO: Remove
```

Comment:
@israel-hdez This was not removed prior to merge. (also below).

Reply:
Yes, it is intentional. It's a note to remove this line once we have a new design... Perhaps my note it's too short and needed more explanation :/

Reply:
Yes, too short :)